go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/Mic92/sops-nix
 
-go 1.18
+go 1.22
+
+toolchain go1.23.3
```

Comment:
Please drop that otherwise go while whine about toolchaik incompatibles and create useless friction.